### ISSUE TITLE
no rollup config file installed in v3.1.1 also not able to publish to npm

### ISSUE THREAD

**Issue by btanwar** [transitive-bullshit/create-react-library#276]

**Comment by psyycker** [comment]
Hi, the V3 is not using rollup anymore by a package called microbundle. It doesn't need any configuration.
About your public problem, maybe you could give us a tiny bit more details?

**Comment by DominickVale** [comment]
Yep, that's the way to go. Make it impossible to us to configure and stop maintaining the repository at the same time, so that we are effectively locked up on our issues. Nice.

**Comment by psyycker** [comment]
I know and I'm not for that, we should be able to configure freely our library. I never approved those changes and it makes the package quite useless then... Maybe it's time to create another one?
